pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -307,7 +309,7 @@ public synchronized void start() throws PulsarServerException {
                             pulsar.getConfiguration().getDefaultNumberOfNamespaceBundles());
 
             tableview = createServiceUnitStateTableView();
-            tableview.start(pulsar, this::handleEvent, this::handleExisting);
+            tableview.start(pulsar, this::handleEvent, this::handleExisting, this::handleSkippedEvent);
```

Comment:
I am actually not sure if this skip logic is required as this can return deferred lookups too soon.

Reply:
Let me revert this change.

This skip logic is not required. I restored this logic just because I want to handle the skipped Free events in the same method. But from your explanation [here](https://github.com/apache/pulsar/pull/23349#discussion_r1775957883), handling the skipped Free events might be dangerous so I'm going to think of another solution